apps/web/pages/_app.tsx

```diff
@@ -44,6 +47,10 @@ function MyApp(props: AppProps) {
       <Head>
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
       </Head>
+      <Script
+        id="calcom-version"
+        dangerouslySetInnerHTML={{ __html: `window.CalComVersion = '${calComVersion}'` }}
+      />
```

Comment:
how would the user know the version then? they would need to log it themselves, right?

Reply:
Yeah, this is mostly for debugging. But if we can identify a place to show on all pages(public/non public), we can show it in UI as well